src/bin/setup.rs

```diff
@@ -23,6 +23,7 @@ async fn main() {
             r#"
 id = {id}
 rpc_address=0.0.0.0:{rpc_port}
+rocksdb_dir=.rocks{i}
```

Comment:
I think this should be `nodes/id/1/.rocks`. All data for a particular node should go into the node directory.

Reply:
Let us also delete the existing .rocks directory each time we run `setup` (can customize this later, but for now, I think we don't want to preserve any state across runs)